posthog/api/cohort.py

```diff
@@ -280,17 +279,14 @@ def insert_cohort_actors_into_ch(cohort: Cohort, filter_data: Dict):
     elif insight_type == INSIGHT_STICKINESS:
         stickiness_filter = StickinessFilter(data=filter_data, team=cohort.team)
         entity = get_target_entity(stickiness_filter)
-        query_builder = ClickhouseStickinessActors(cohort.team, entity, stickiness_filter)
+        query_builder = StickinessActors(cohort.team, entity, stickiness_filter)
     elif insight_type == INSIGHT_FUNNELS:
         funnel_filter = Filter(data=filter_data, team=cohort.team)
-        if funnel_filter.correlation_person_entity:
```

Comment:
Hmm, does this come back in a future PR? Because without this, correlation cohorts don't work?

Reply:
check out get_funnel_actor_class. changed that to include this condition with an extra EE_available check